lib/src/pages/splash.dart

```diff
@@ -16,12 +17,8 @@ class SplashScreen extends StatelessWidget {
 
     Future.delayed(const Duration(seconds: 2), () async {
       bool isLoggedIn = await UserValidation().checkUserLogin();
-      Navigator.of(context).push(
-        MaterialPageRoute(
-            builder: (context) => const HomePage(title: 'Help a Paw'),
-            settings: RouteSettings(
-                name: isLoggedIn ? global.homeRoute : global.loginRoute)),
-      );
+
+      Navigator.of(context).pushNamed(isLoggedIn ? AppRoutes.homeScreen : AppRoutes.loginRoute);
```

Comment:
Actually we want to always lend on the home screen and not force users to login.

Reply:
done